starter/src/main/java/com/vivy/springdoc/OpenAPIAutoConfiguration.java

```diff
@@ -1,18 +1,34 @@
 package com.vivy.springdoc;
 
+import io.swagger.v3.oas.models.Components;
 import io.swagger.v3.oas.models.OpenAPI;
+import io.swagger.v3.oas.models.security.SecurityRequirement;
+import io.swagger.v3.oas.models.security.SecurityScheme;
+import org.springdoc.core.SwaggerUiConfigParameters;
+import org.springframework.boot.autoconfigure.AutoConfigureBefore;
 import org.springframework.boot.autoconfigure.condition.ConditionalOnMissingBean;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Configuration;
-
+import org.springframework.context.annotation.PropertySource;
+import java.util.List;
 
 @Configuration
+@PropertySource("classpath:swagger-ui.properties")
+@AutoConfigureBefore(SwaggerUiConfigParameters.class)
 public class OpenAPIAutoConfiguration {
 
     @Bean
     @ConditionalOnMissingBean
-    public OpenAPI openAPI() {
-        return new BearerOpenAPI();
+    public OpenAPI bearerOpenAPI() {
+        return new OpenAPI().components(
+                new Components().addSecuritySchemes("bearer-jwt",
```

Comment:
```suggestion
                new Components().addSecuritySchemes(
                        "bearer-jwt",
```
codestyle: after open `(` if you have line break later, shouldn't be arguments on the same line